backwards-compatability/node10/package.json

```diff
@@ -8,8 +8,8 @@
     "test:backcompat": "tsc --noEmit index.ts"
   },
   "dependencies": {
-    "@opentelemetry/sdk-node": "^0.17.0",
-    "@opentelemetry/tracing": "^0.17.0"
+    "@opentelemetry/sdk-node": "0.17.0",
```

Comment:
Hmm, this seems to be strange. As far as I know we want to pin dev dependencies but not prod dependencies.

Reply:
I think it pins dependencies outside the monorepo by default

Reply:
but lerna.json tells that `backwards-compatability/*` is part of the project.

Reply:
Not sure if it's smart enough to read the lerna config. It might just be looking in the packages directory.